README.md

```diff
@@ -194,6 +194,13 @@ Adding the `toggle-btn` class to your radio inputs converts them to toggle butto
 <input type="radio" class="sbtn toggle-btn" />
 ```
 
+## Rounded buttons Radius
+To add the rounded buttons with a particular radius, you can use various classes like `rounded-x-btn` where x is the radius in pixels which you want.
+
+```html
+<button class="rounded-5-btn">Rounded button </button>
+```
+
```

Comment:
Please specify the range of x (can be from 5 till ...)

Reply:
Hello, @shahednasser  I have worked on the requested changes. Please review them.